tests/model_fields/test_uuid.py

```diff
@@ -35,6 +35,14 @@ def test_null_handling(self):
         self.assertEqual(loaded.field, None)
 
 
+class TestMigrations(TestCase):
```

Comment:
Trying to basically do `assertNumQueries(0)` on the class would probably be a starting point. Not sure if this would add much overhead.